frontend/src/pages/users/admin/UserAdmin.jsx

```diff
@@ -1,26 +1,37 @@
 import App from "../../../App";
 import UserEmailComboBox from "../../../components/Users/UserEmailComboBox/index.js";
-import { useGetDivisionsQuery } from "../../../api/opsAPI.js";
+import React from "react";
+import UserInfo from "../../../components/Users/UserInfo/UserInfo.jsx";
```

Comment:
don't need the double UserInfo on the import

`../../../components/Users/UserInfo"`